### PR TITLE
Begone Thot Command

### PR DESCRIPTION
**Pull request by Connor-Harkness**
Initial commit and modifications

### CODE REVIEW

**Comment by Connor-Harkness** [comment]
Before you pull this, might want to uhh. uncomment the check for has_any_role('Staff') cause im an idiot and left it commented

File: cogs/Bangandthethotisgone.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    # Yuki#2520 (190007233919057920)
+    # Tormund#3852 (238616410941554688)
+    # ZamieltheHunter#4105 (182287810173206529)
+    # Gala#8207 (73389450113069056)
+    # Riiiina-Chan#9369 (265599587333570563)
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    with open('data.json') as Json_file:
```
Comment (Nekorooni) [inline cogs/Bangandthethotisgone.py]: Shouldn't this be under init? >.<

Comment (Connor-Harkness) [reply cogs/Bangandthethotisgone.py]: Probably o3o

**Comment by Connor-Harkness** [comment]
Tell you what, lemme close this then reopen one with these changes

**Pull request closed without merging**